tests/vmi_configuration_test.go

```diff
@@ -165,7 +165,16 @@ var _ = Describe("Configurations", func() {
 						kubev1.ResourceMemory: resource.MustParse("64M"),
 					},
 				}
-				vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				// Retrying 5 times with its interval set to one second, then if you still succeeds
+				// in VMI creation, things must be going wrong.
+				for i := 0; i < 5; i++ {
+					vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+					if err != nil {
+						break
+					}
+					virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Delete(vmi.Name, &metav1.DeleteOptions{})
+					time.Sleep(1 * time.Second)
```

Comment:
@lkpdn you can use something like this:

```golang
Eventually(func() {
    ....
}, 5 * time.Second, 1 * time.Second).Should(Equal(something))
```

Reply:
Thanks, but now I'm rather in favor of your original suggestion, i.e. no sleep, just trying five times.

Reply:
Hmm, testing several times with 5 times with no sleep, seems that we better to insert sleep to rigidly testing "what we want to test". So I'll push with `Eventually(func()...`.

Reply:
Sounds good.